SupportClient/SupportClient/Extensions/UIButton+Extension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIButton+Extension.swift
```

Comment:
Can we rename this to `MaskedButton.swift` Idt this would fall under extensions per say

Reply:
I deleted this file on my local repo, but none of my file deletions were pushed to GitHub...